lib/src/base-styles/_nx-pagination.scss

```diff
@@ -38,35 +45,34 @@
   }
 }
 
-.nx-btn--pagination {
+.nx-btn.nx-btn--pagination {
   @include nx-text-helpers.regular;
-  background-color: transparent;
   border-color: transparent;
   color: var(--nx-color-text);
 
   &:hover {
-    background-color: transparent;
-    border-color: var(--nx-color-interactive-border-hover);
+    background-color: var(--nx-color-interactive-background-hover);
+    color: var(--nx-color-text-dark);
   }
 
-  &:focus {
-    border-color: var(--nx-color-interactive-border-focus);
+  &:focus:not(:active) {
+    outline: 2px solid var(--nx-color-interactive-border-focus);
+    outline-offset: -2px;
   }
 
   &:active {
-    background-color: var(--nx-swatch-indigo-97);
-    border-color: var(--nx-color-interactive-border-hover);
-    box-shadow: none;
+    background-color: var(--nx-swatch-grey-97);
     color: var(--nx-color-text);
   }
 
   &.selected {
-    background-color: var(--nx-swatch-indigo-15);
+    background-color: var(--nx-swatch-blue-40);
     color: var(--nx-color-component-background);
     cursor: default;
 
     &:focus {
-      border-color: var(--nx-color-interactive-border-focus);
+      outline: 2px solid var(--nx-color-component-background);
+      outline-offset: -5px;
```

Comment:
I think we should create a placeholder for the outline since it's being used a few times.

Reply:
> I think we should create a placeholder for the outline since it's being used a few times.

Sounds good! Added in here: [b83a041](https://github.com/sonatype/sonatype-react-shared-components/pull/1004/commits/b83a041b9d981aec94b6707c4f9494d1f5d2f011)